var/spack/repos/builtin/packages/mpich/package.py

```diff
@@ -59,7 +59,7 @@ class Mpich(AutotoolsPackage, CudaPackage, ROCmPackage):
 
     depends_on("c", type="build")  # generated
     depends_on("cxx", type="build")  # generated
-    depends_on("fortran", type="build")  # generated
+    depends_on("fortran", type="build", when="+fortran")
```

Comment:
Does this change the default to not require Fortran? That's not desirable. Fortran is opt-out, not opt-in.

Reply:
It does not. The variant is untouched and still defaults to `True`.

Reply:
OK, thanks for confirming. This change LGTM.